lib/koala/api/graph_collection.rb

```diff
@@ -16,30 +16,41 @@ class GraphCollection < Array
         attr_reader :api
         # The entire raw response from Facebook.
         attr_reader :raw_response
+        # The headers from the Facebook response
+        attr_reader :headers
 
         # Initialize the array of results and store various additional paging-related information.
         #
-        # @param response the response from Facebook (a hash whose "data" key is an array)
+        # @param [Koala::HTTPService::Response] response object wrapping the raw Facebook response
         # @param api the Graph {Koala::Facebook::API API} instance to use to make calls
         #            (usually the API that made the original call).
         #
         # @return [Koala::Facebook::API::GraphCollection] an initialized GraphCollection
         #         whose paging, summary, raw_response, and api attributes are populated.
         def initialize(response, api)
-          super response["data"]
-          @paging = response["paging"]
-          @summary = response["summary"]
-          @raw_response = response
+          super response.data["data"]
+          @paging = response.data["paging"]
+          @summary = response.data["summary"]
+          @raw_response = response.data
```

Comment:
Glad that it works as is for you. I don't see how it is a breaking change from the API level though. I think everything should work as is? Unless people are directly calling `GraphCollection.evaluate`?